Cargo.toml

```diff
@@ -40,7 +40,6 @@ rust-version = "1.75"
 version = "0.3.0"
 
 [workspace.dependencies]
```

Comment:
All `iceoryx2-**` dependencies must stay in the workspace dependencies section. The idea here is that all other packages reference them like `iceoryx2-** = { workspace = true }`.

Reply:
Currently they are not referenced, therefore, `cargo udeps` removed them. As long as they are used in another member crate they are not removed by `cargo udeps`. Technically they are not needed. 

The complete list of members is still available in `members`.

Reply:
Mmmmhhh maybe I was wrong. If the CI compiles and all tests are still running then it is fine with me.